src/Expr.js

```diff
@@ -143,6 +143,13 @@ var exprToString = function(expr, caller) {
   var keys = Object.keys(expr)
   var fn = keys[0]
 
+  // Versioned queries/lambdas will have an api_version field.
+  // We want to prevent it from being parsed and displayed as:
+  // Query(ApiVersion("3", "X", Var("X")))
```

Comment:
please, add a test on https://github.com/fauna/faunadb-js/blob/master/test/values.test.js#L146 to make sure this is correct

Reply:
@marrony Would this test suffice? https://github.com/fauna/faunadb-js/pull/308/files#diff-c2e271250d0fe43a2f2478970f73543eR2713

Reply:
since you mentioned it, that test doesn't make sense in this context because it's not deserializing anything, it would make more sense if you move it to "pretty print" on `values.test.js`